### DISCUSSION TITLE
Free Radial Expansion BC - DisplacementAboutAxis

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#25860] [Q&A Modules: Solid mechanics]
Is it possible to use the DisplacementAboutAxis BC to restrict a body to only radial/axial expansion? I assumed that I could do something similar to the following, but it just makes the displacements in x and y zero on the face.

```
  [xdir]
    type = DisplacementAboutAxis
    boundary = 'back'
    function = 0
    angular_velocity = false
    angle_units = degrees
    axis_origin = '0. 0. 0.'
    axis_direction = '0. 0. 1.'
    component = 0
    variable = 'disp_x'
    displacements = 'disp_x disp_y disp_z'
  []
  [ydir]
    type = DisplacementAboutAxis
    boundary = 'back'
    function = 0
    angular_velocity = false
    angle_units = degrees
    axis_origin = '0. 0. 0.'
    axis_direction = '0. 0. 1.'
    component = 1
    variable = 'disp_y'
    displacements = 'disp_x disp_y disp_z'
  []
```

**Comment by GiudGiud** [comment]
@backmari @arovinelli

**Comment by backmari** [comment]
I haven't used this in a long time, but the original purpose of the BC was to fix one end of a cylinder and twist the other end. See this test for an example: modules/tensor_mechanics/test/tests/torque_reaction/torque_reaction_cylinder.i

**Reply by ghost** [reply]
So it really isn't meant to be used to prevent a cylinder from twisting but more to prescribe components of a torque?

**Reply by backmari** [reply]
Yes

**Reply by ghost** [reply]
ok thanks
